src/makeRobotAccountant.js

```diff
@@ -7,6 +7,19 @@
 
 function makeRobotAccountant() {
   // write code here
+  let callCount = 0;
+
+  return (firstValue) => {
+    callCount++;
+
+    const condition = !(callCount % 2) && callCount > 3;
```

Comment:
What is `condition` mean? Which condition is it? Use more meaningful variable names (for example `hasError`, `isBroken`, etc.)